datahub-web-react/src/app/search/AdvancedFilterSelectValueModal.tsx

```diff
@@ -0,0 +1,211 @@
+import React from 'react';
+import { FacetMetadata, EntityType } from '../../types.generated';
+import { SelectContainerModal } from '../entity/shared/containers/profile/sidebar/Container/ContainerSelectModal';
+import { SetDomainModal } from '../entity/shared/containers/profile/sidebar/Domain/SetDomainModal';
+import { EditOwnersModal } from '../entity/shared/containers/profile/sidebar/Ownership/EditOwnersModal';
+import { SelectPlatformModal } from '../entity/shared/containers/profile/sidebar/Platform/SelectPlatformModal';
+import EditTagTermsModal from '../shared/tags/AddTagsTermsModal';
+import { ChooseEntityTypeModal } from './ChooseEntityTypeModal';
+import { EditTextModal } from './EditTextModal';
+
+type Props = {
+    facet?: FacetMetadata | null;
+    filterField: string;
+    onSelect: (values: string[]) => void;
+    onCloseModal: () => void;
+    initialValues?: string[];
+};
+
+export const AdvancedFilterSelectValueModal = ({
+    filterField,
+    onSelect,
+    onCloseModal,
+    initialValues,
+    facet,
+}: Props) => {
+    if (filterField === 'owners') {
+        return (
+            <EditOwnersModal
+                title="Select Owners"
+                urns={[]}
+                defaultValues={initialValues?.map((urn) => ({
```

Comment:
nit: might be easier to pull this out into its own const to improve readability

Reply:
same with some of the other inline things defined in this component

Reply:
They are all slightly different per modal, i think it might be simpler just to render them inline?

Reply:
fair enough that's okay with me